### PR TITLE
Documentation updates before release

### PR DESCRIPTION
**Pull request by akolotov**
The main README and deployment README are updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,17 +6,21 @@ two EVM-based blockchain networks by collecting bridge validators signatures to
...
-* to swap a token presented by an ERC20 contract in Home network to an existing ERC20 token in Foreign network.
+* to swap a token presented by an existing ERC20 contract in Foreign network to an ERC20 token in Home network, one pair of bridge contracts correspond to one pair of ERC20 tokens.
 
-POA bridge consist of several main parts:
-* Home Bridge is a smart contract that should be deployed in POA.network;
+This version of contract is intended to be work with [the bridge process implemented on NodeJS](https://github.com/poanetwork/bridge-nodejs).
+Please refer to the bridge process documentation to deploy it and configure.
+
+POA bridge contracts consist of several main parts:
+* Home Bridge is a smart contract that should be deployed in POA.Network;
```
Comment (andogro) [inline README.md]: **The** Home Bridge

File: README.md [hunk 2]
```diff
@@ -6,17 +6,21 @@ two EVM-based blockchain networks by collecting bridge validators signatures to
...
 * Foreign Bridge is a smart contract that should be deployed in Ethereum Mainnet;
```
Comment (andogro) [inline README.md]: **The** Foreign Bridge

File: README.md [hunk 3]
```diff
@@ -6,17 +6,21 @@ two EVM-based blockchain networks by collecting bridge validators signatures to
...
 * Depending on type of relay operations the following components are used as well:
   * in `NATIVE-TO-ERC` mode: ERC20 token (in fact, ERC677 extension is used) should be deployed on Foreign network;
```
Comment (andogro) [inline README.md]: * in `NATIVE-TO-ERC` mode: **The** ERC20 token (in fact, ERC677 extension is used) should be deployed on **the** Foreign network;

File: README.md [hunk 4]
```diff
@@ -6,17 +6,21 @@ two EVM-based blockchain networks by collecting bridge validators signatures to
...
   * in `ERC-TO-ERC` mode: ERC20 token (in fact, ERC677 extension is used) should be deployed on Home network;
```
Comment (andogro) [inline README.md]: * in `ERC-TO-ERC` mode: **The** ERC20 token (in fact, ERC677 extension is used) should be deployed on **the** Home network;

File: README.md [hunk 5]
```diff
@@ -6,17 +6,21 @@ two EVM-based blockchain networks by collecting bridge validators signatures to
...
+* Validators is a smart contract that should be deployed in both POA.Network and Ethereum Mainnet.
```
Comment (andogro) [inline README.md]: Validators is a smart contract that should be deployed in both **the** POA.Network and **the** Ethereum Mainnet.

File: README.md [hunk 6]
```diff
@@ -25,12 +29,13 @@ Responsibilities and roles of the bridge:
...
-  - listen for Withdraw events on Foreign bridge to unlock funds on Home Bridge
+  - listen for `UserRequestForSignature` events on Home Bridge and sign an approval to relay assets on Foreign network;
+  - listen for `CollectedSignatures` events on Home Bridge as soon enough signatures are collected and transfer all collected signatures to Foreign Bridge contact;
+  - listen for `UserRequestForAffirmation` or `Transfer` (depending on the bridge mode) events on Foreign Bridge and send approval to Home Bridge to relay assets from Foreign Network to Home
 - User role:
-  - sends POA coins to Home bridge in order to receive ERC20 token on Foreign Bridge
-  - sends ERC20 POA20 token on Foreign Bridge in order to receive POA coins on Home Bridge
-
+  - sends assets to Bridge contracts:
+    - in `NATIVE-TO-ERC` mode: send native coins to Home Bridge to receive ERC20 tokens from Foreign Bridge, send ERC20 tokens to Foreign Bridge to unlock native coins from Home Bride;
```
Comment (andogro) [inline README.md]: - in `NATIVE-TO-ERC` mode: send native coins to **the** Home Bridge to receive ERC20 tokens from **the** Foreign Bridge, send ERC20 tokens to **the** Foreign Bridge to unlock native coins from **the** Home **Bridge**;

File: README.md [hunk 7]
```diff
@@ -25,12 +29,13 @@ Responsibilities and roles of the bridge:
...
+    - in `ERC-TO-ERC` mode: transfer ERC20 tokens to Foreign Bridge to mint ERC20 tokens on Home Network, transfer ERC20 tokens to Home Bridge to unlock ERC20 tokens on Foreing networks.
```
Comment (andogro) [inline README.md]: - in `ERC-TO-ERC` mode: transfer ERC20 tokens to **the** Foreign Bridge to mint ERC20 tokens on **the** Home Network, transfer ERC20 tokens to **the** Home Bridge to unlock ERC20 tokens on **Foreign** networks.

File: README.md [hunk 8]
```diff
@@ -40,60 +45,6 @@ npm install
 # To Deploy
 Check README.md in `deploy` folder
```
Comment (andogro) [inline README.md]: Check **the** README.md in `deploy` folder

File: deploy/README.md [hunk 9]
```diff
@@ -8,79 +8,207 @@ npm run compile
 2. create `.env` file
 `cp .env.example .env`
 
-3. if it is necessary to have the bridge to relay native coins to an erc20 token, the following example could be used for `.env` file
+3. If it is necessary, deploy and configure a multi-sig wallet contract which will
+   be used to manage the bridge contracts after deployment.
+
+4. adjust parameters in `.env` file as per recommendation below depending on the desired bridge mode
```
Comment (andogro) [inline deploy/README.md]: adjust **the** parameters in **the** `.env` file ~~as per recommendation below~~ depending on the desired bridge mode <see recommendation below>

File: deploy/README.md [hunk 10]
```diff
@@ -8,79 +8,207 @@ npm run compile
 2. create `.env` file
 `cp .env.example .env`
 
-3. if it is necessary to have the bridge to relay native coins to an erc20 token, the following example could be used for `.env` file
+3. If it is necessary, deploy and configure a multi-sig wallet contract which will
+   be used to manage the bridge contracts after deployment.
+
+4. adjust parameters in `.env` file as per recommendation below depending on the desired bridge mode 
+
+5. Fill balance of the deployment account in Home and Foreign networks
```
Comment (andogro) [inline deploy/README.md]: Fill **the** balance of the deployment account in Home and Foreign networks

File: deploy/README.md [hunk 11]
```diff
@@ -8,79 +8,207 @@ npm run compile
...
+
+4. adjust parameters in `.env` file as per recommendation below depending on the desired bridge mode 
+
+5. Fill balance of the deployment account in Home and Foreign networks 
+
+6. Run `node deploy.js` 
+
+## Configuration for `NATIVE-TO-ERC` Bridge mode  
+
+Here is an example of `.env` file for `native-to-erc` bridge mode.
```
Comment (andogro) [inline deploy/README.md]: Here is an example of **an** `.env` file for `native-to-erc` bridge mode.

File: deploy/README.md [hunk 12]
```diff
@@ -8,79 +8,207 @@ npm run compile
...
+# The type of bridge. Defines set of contracts to be deployed.
 BRIDGE_MODE=NATIVE_TO_ERC
 
+# The private key hex value of the account responsible for contracts
+# deployments and initial configuration. The account's balance must contain
+# funds from both networks.
 DEPLOYMENT_ACCOUNT_PRIVATE_KEY=67..14
+# The "gas" parameter set in every deployment/configuration transaction.
 DEPLOYMENT_GAS_LIMIT=4000000
+# The "gasPrice" parameterset in every deployment/configuration transaction on
```
Comment (andogro) [inline deploy/README.md]: parameter set (add space)

File: deploy/README.md (hunk 11 above)
Comment (andogro) [inline deploy/README.md]: I think this information would be easier to view in a table format, similar to what exists currently on the BridgeUI site. I can make a suggestion to convert in my PR if you would prefer to not change at the moment.

File: README.md [hunk 13]
```diff
@@ -40,60 +45,6 @@ npm install
...
-- address of Foreign Bridge **Validators** address Proxy ( step# 10) `0x06`
-- address of POA20 token contract. Step#9 `0x05`
-- Daily Limit in wei: Example `1000000000000000000` == 1 ether
-- Maximum Per Transaction Limit in wei: Example `100000000000000000` == 0.1 ether. Should be less than Daily Limit
-- Minimum Per Transaction Limit in wei: Example `10000000000000000` == 0.01 ether. Should be less than Daily Limit and less than MaxPerTx
-18. Call `transferOwnership` of POA20 contract(0x05) with 1 parameter:
-- address of Foreign Bridge Proxy (`0x08`) from step#14
-
-
 # To Flatten
```
Comment (andogro) [inline README.md]: Can we add a #Testing heading with any information about testing here - ie the testing command, any additional flags, etc.

File: deploy/README.md (hunk 11 above)
Comment (akolotov) [reply deploy/README.md]: I am not sure how to address correctly the fact that some parameters have different meaning for different bridge modes.

**Pull request merged**